app/helpers/host_helper/textual_summary.rb

```diff
@@ -235,6 +235,14 @@ def textual_cluster
     h
   end
 
+  def textual_physical_server
+    if !@record.physical_server_id.nil?
+      {:label => _("Physical Server"), :value => @record.physical_server.try(:name), :icon => "pficon pficon-server", :link => url_for(:controller => 'physical_server', :action => 'show', :id => @record.physical_server_id)}
+    else
+      {:label => _("Physical Server"), :value => "None", :icon => "pficon pficon-server"}
+    end
+  end
+
```

Comment:
Flip the order here that way we do not have to use `!` (negative conditional)
Also note the use of `_("None")` which takes care of i18n.
```ruby
if @record.physical_server_id.nil?
  {:label => _("Physical Server"), :value => _("None"), :icon => "pficon pficon-server"}
else
  {:label => _("Physical Server"), :value => @record.physical_server.try(:name), :icon => "pficon pficon-server", :link => url_for(:controller => 'physical_server', :action => 'show', :id => @record.physical_server_id)}
end
```